portfolio/src/main/java/com/google/sps/data/Comments.java

```diff
@@ -14,20 +14,21 @@
 
 package com.google.sps.data;
 import java.util.ArrayList;
+import java.util.HashMap;
 
 /** Class containing comment information. */
 
 public final class Comments {
 
-  // List of comments
-  private final ArrayList<String> comments = new ArrayList<>();
+  // List of comments and their authors
+  private final HashMap<String, String> comments = new HashMap<String, String>();
   
```

Comment:
you don't need tyep declaration on the right hand side.
Also, prefer to use the interface `Map` rather than the specific type `HashMap` for the variable type

```
private final Map<String, String> comments = new HashMap<>
```